### PR TITLE
configure sentry

### PR DESCRIPTION
**Pull request by rolandpo**
configure sentry + some cleanup

### CODE REVIEW

File: dev.env [hunk 1]
```diff
@@ -16,6 +16,7 @@ NEXT_PUBLIC_GITHUB=https://api.github.com
 
 MIXPANEL_ID=
 GA_ID=
+SENTRY_DSN=
```
Comment (shahthepro) [inline dev.env:19]: Let's make sure to set this env before we deploy

File: sentry.server.config.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN
+const NODE_ENV = process.env.NODE_ENV
+
+Sentry.init({
+  dsn: SENTRY_DSN || 'https://67fc84f1cafe443c8bb9d73eb172712a@o225462.ingest.sentry.io/4504001045790720',
```
Comment (shahthepro) [inline sentry.server.config.js:11]: Question: Should we use a different DSN for Next.js client and Next.js server?

Comment (franckc) [reply sentry.server.config.js:11]: Discussed with Roland. We decided to keep it simple for now and use a single DSN. Then we can improve afterwards.

**Review by franckc — APPROVED** [review]
LGTM

**Review by franckc — APPROVED** [review]
LGTM

**Pull request merged**